config/pluginspec.yaml

```diff
@@ -211,516 +315,868 @@ procedures:
+      For example, <code>MySqlDS</code>.
+    adoc: |
+      Name of the datasource.
+      This also serves as the unique identifier for this resource.
+      For example, `MySqlDS`.
+    label: "Datasource name"
   - name: "jndiName"
     type: "entry"
     required: true
-    documentation: "JNDI Name of the Data Source. For example java:/MySqlDS"
-    label: "JNDI Name:"
+    htmlDocumentation: |
+      JNDI name of the datasource.
+      For example, <code>java:/MySqlDS</code>.
+    adoc: |
+      JNDI name of the datasource.
+      For example, `java:/MySqlDS`.
+    label: "JNDI name"
   - name: "jdbcDriverName"
     type: "entry"
     required: true
-    documentation: "Defines the JDBC driver the data source should use. It is a\
-      \ symbolic name matching the name of installed driver. For example mysql"
-    label: "JDBC Driver Name:"
+    htmlDocumentation: |
+      Defines the JDBC driver the datasource should use.
+      It is a symbolic name matching the name of installed driver.
+      For example, <code>mysql</code>.
+    adoc: |
+      Defines the JDBC driver the datasource should use.
+      It is a symbolic name matching the name of installed driver.
+      For example, `mysql`.
+    label: "JDBC driver name"
   - name: "connectionUrl"
     type: "entry"
     required: false
-    documentation: "Required for JBoss EAP 6.X, 7.0. Not Required for 7.1. The JDBC\
-      \ driver connection URL. For example jdbc:mysql://localhost:3306/mysqldb"
-    label: "Connection URL:"
+    htmlDocumentation: |
+      The JDBC driver connection URL.
+      For example, <code>jdbc:mysql://localhost:3306/mysqldb</code>.
+      This is required for JBoss EAP 6.X and 7.0; it is not required for 7.1.
+    adoc: |
+      The JDBC driver connection URL.
+      For example, `\jdbc:mysql://localhost:3306/mysqldb`.
+      This is required for JBoss EAP 6.X and 7.0; it is not required for 7.1.
+    label: "Connection URL"
   - name: "dataSourceConnection_credential"
     type: "credential"
     required: true
-    documentation: "Credential that contains username and password used for the\
-      \ connection (need to be attached to step)."
-    label: "Data Source Connection Credentials:"
+    htmlDocumentation: |
+      Credentials that contain the username and password used for the connection, and need to be attached to the step.
+    adoc: |
+      Credentials that contain the username and password used for the connection, and need to be attached to the step.
+    label: "Datasource connection credentials"
   - name: "enabled"
     type: "entry"
     required: true
-    documentation: "If the created Data Source should be enabled or not. Default\
-      \ is true."
-    label: "Enabled:"
+    htmlDocumentation: |
+      If selected, the created datasource is enabled.
+    adoc: |
+      If selected, the created datasource is enabled.
+    label: "Enabled"
     checkedValue: "1"
     uncheckedValue: "0"
```

Comment:
I think this parameter is incorrect. I updated the text assuming it was a checkbox based on the presence of `checkedValue` and `uncheckedValue`. However, looking at the procedure, this is actually a text field and no default value is specified.
<img width="617" alt="image" src="https://github.com/electric-cloud/EC-JBoss/assets/80066741/af821b4e-0960-4717-a5c2-7182033469c1">